homeassistant/components/glances/__init__.py

```diff
@@ -47,6 +48,7 @@
             vol.Optional(CONF_SSL, default=False): cv.boolean,
             vol.Optional(CONF_VERIFY_SSL, default=True): cv.boolean,
             vol.Optional(CONF_VERSION, default=DEFAULT_VERSION): vol.In([2, 3]),
+            vol.Optional(CONF_ACCEPT_UNAVAILABLE, default=False): cv.boolean,
```

Comment:
I don't think this should be a config option. We should decide on a behavior that is acceptable for a majority of users and stick to that.

Reply:
Ideally we should only print the log statement once when unavailable to help stop the log flooding.

Reply:
For such cases I think the user can change the logging level to `fatal` or `critical` for the glances integration to skip logging these errors.

Reply:
indeed, problem is logger ignores it. my conf:
logger:
  default: warn
  logs:
    homeassistant.components.glances: fatal
still logs error messages, and not debug

Reply:
I did some testing. Apparently the error is coming from `glances_api` and not the Glances integraion. This is [the error](https://github.com/fabaff/python-glances-api/blob/6f28cf570d575dba18948e0cf307941b1c088a66/glances_api/__init__.py#L49) that is showing in the logs. I am not sure if we can control the logging level for this directly.

Reply:
It's not acceptable to pass a custom logger to the api, since that's not how logging should be handled. The library should use a logger per module. The the app can filter that as needed. That seems already to be the case here.

Reply:
Okay, i got it now. 
2019-10-24 11:08:34 ERROR (MainThread) [glances_api] Can not load data from Glances API
refers to glances_api root component rather than homeassistant.components.glances
I think we should include an integration example in Logger docs, rather than homeassistant.* and aiohttp only to prevent further confusion.
E.g. Do note the difference between glances_api and homeassistant.components.glances, or specify that
TIMESTAMP LEVEL [logger] *logs*
should be placed in
```yaml
logger:
  default: warn
  logs:
    [logger]: fatal
```

Reply:
Yes! If you have time please make a PR to the docs to clarify. Thanks!